dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/GenericFilter.java

```diff
@@ -107,6 +108,16 @@ public Result invoke(Invoker<?> invoker, Invocation inv) throws RpcException {
                                             args[i].getClass().getName());
                         }
                     }
+                }else if (ProtocolUtils.isProtobufGenericSerialization(generic)) {
```

Comment:
pay more attention to code style. You should import https://github.com/apache/incubator-dubbo/tree/master/codestyle and go through your change thoroughly.

Reply:
done， test by format another file,it works well